### ISSUE TITLE
announcement: new activity has moved to the AndroidHardening organization

### ISSUE THREAD

**Issue by thestinger** [AndroidHardeningArchive/bugtracker#868]
If you're still watching this repository, you should be aware that new activity has moved to https://github.com/AndroidHardening. The new manifest for future work on AOSP privacy and security improvements is https://github.com/AndroidHardening. I won't be using this repository for issue tracking anymore as this is a fresh start without baggage associated with the legacy projects.

To clarify what has happened, Copperhead is no longer associated with my open source work on privacy and security. My former business partner hijacked the company and pushed me out. He attempted to seize control and ownership over my projects, which I rejected. I responded to the infrastructure being seized and threats of seizing my signing keys by wiping them and ending the previous incarnation of the project to keep users from being compromised. The company was supposed to be supporting my independent open source work, and my business partner no longer liked the arrangement as he wanted to make changes counter to the goals of the project and user privacy / security which were blocked by the company not controlling it. I could not in good conscious hand over control of the project people had placed their trust in and instead brought it to an end and moved the repositories to this AndroidHardeningArchive repository.

Copies of the repositories uploaded by Copperhead are *not* the original projects and they *do not own the code* nor do they have a license to use the code commercially from me. They convinced me to use non-commercial usage license but never had to the foresight to open a license from me and they lost any claim to being allowed to use it by pushing me out and destroying years of what I had worked to build.

**Comment by thestinger** [comment]
The new issue tracker is https://github.com/AndroidHardening/os_issue_tracker.

**Comment by thestinger** [comment]
The new issue tracker is https://github.com/AndroidHardening/os_issue_tracker/issues.

**Comment by thestinger** [comment]
The new issue tracker is https://github.com/GrapheneOS/os_issue_tracker/issues.
